CommandDotNet/TypeDescriptors/ComponentModelTypeDescriptor.cs

```diff
@@ -19,7 +20,9 @@ public string GetDisplayName(ArgumentInfo argumentInfo)
 
         public object ParseString(ArgumentInfo argumentInfo, string value)
```

Comment:
would the code in this method make more sense if the method name was changed to ParseSingleValue?